docs/sources/next/javascript-api/k6-experimental/browser/page/innertext.md

```diff
@@ -26,9 +26,9 @@ Returns the `element.innerText`.
 
 ### Returns
 
-| Type   | Description                   |
-| ------ | ----------------------------- |
-| string | The innerText of the element. |
+| Type              | Description                                                 |
+| ----------------- | ----------------------------------------------------------- |
+| `Promise<string>` | A Promise that fullfils with the inner text of the element. |
```

Comment:
Users can get what "inner text" means. I can change this if you have a strong opinion.

Reply:
I think having it as `innerText` instead of `inner text` would make things a tiny bit clearer. Happy to wait for others to chime in or it can be ignore for now.

Reply:
@heitortsergent WDYT?

Reply:
Updated here [5e945c6](https://github.com/grafana/k6-docs/pull/1602/commits/5e945c6196d6b3d7b0ed6f1ae79ff728c3add3a3).